vita3k/gui/src/game_selector.cpp

```diff
@@ -15,18 +15,50 @@
 // with this program; if not, write to the Free Software Foundation, Inc.,
 // 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+#include "private.h"
+
 #include <gui/functions.h>
 #include <gui/state.h>
 
-#include "private.h"
-
 #include <host/state.h>
+
+#include <util/log.h>
 #include <util/string_utils.h>
 
 using namespace std::string_literals;
 
 namespace gui {
 
+bool refresh_game_list(GuiState &gui, HostState &host) {
+    auto game_list_size = gui.game_selector.games.size();
+
+    if (!gui.game_selector.games.empty())
+        gui.game_selector.games.clear();
+    if (!gui.game_selector.icons.empty())
+        gui.game_selector.icons.clear();
+
+    if (!gui.game_selector.games.empty() && !gui.game_selector.icons.empty())
```

Comment:
This will never be true. Drop it.

Reply:
Curent code already jave this but i  need change one things

Reply:
Ok? 🤔